### PR TITLE
Feature create add books

### PR DESCRIPTION
**Pull request by rebel216**
Added feature Add and remove Books..

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  type: REMOVE_BOOK,
+  payload,
+});
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
+    case REMOVE_BOOK:
+      return state.filter((book) => book.id !== action.payload);
```
Comment (ShoiraTa) [inline src/redux/books/books.js]: - [ ] Good implementation of updating and deleting books 👍🏻

File: src/redux/configureStore.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
+
+const store = createStore(reducer, applyMiddleware(logger));
```
Comment (ShoiraTa) [inline src/redux/configureStore.js]: - [ ] When starting app with `npm start` I am getting an error, `node_modules/redux-logger doesn't exist`. Kindly make sure that app starts without errors.

File: src/redux/configureStore.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
```
Comment (ShoiraTa) [inline src/redux/configureStore.js]: - [ ] Please add categories reducer as well, currently categories reducer file is empty, it should also have an action and reducer, which should return `under construction`

- [ ] Categories component is missing, please add a category component, The component should contain:
- A button for checking the status

  -  this button would use a function checkCategoriesStatus (name can be different) made in the component (the name may be different)

   - checkCategoriesStatus  dispatches the underConstruction action creator (name of the function can be different)

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @rebel216,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @rebel216,** 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## **Invalid Code Review Request ⛔🚫❌**

You have submitted a project, that _**You have submitted an app that doesn't work, please fix and re-submit again**_

Capture d’écran 2022-09-22 111604


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @rebel216 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Your linters are passing all the checks
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**